cmd/krel/cmd/testgrid.go

```diff
@@ -0,0 +1,269 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"k8s.io/release/pkg/gcp/gcs"
+	"k8s.io/release/pkg/http"
+	"k8s.io/release/pkg/util"
+	"k8s.io/utils/pointer"
+)
+
+type TestGridOptions struct {
+	branch        string
+	boards        []string
+	states        []string
+	bucket        string
+	testgridURL   string
+	renderTronURL string
+}
+
+var testGridOpts = &TestGridOptions{}
+
+type TestgridJobInfo struct {
+	OverallStatus string `mapstructure:"overall_status"`
+	DashboardName string `mapstructure:"dashboard_name"`
+}
+
+type TestGridJob struct {
+	DashboardName string
+	JobName       string
+	Status        string
+	GCSLocation   string
+}
+
+const (
+	statePassing = "PASSING"
+	stateFlaky   = "FLAKY"
+	stateFailing = "FAILING"
+
+	boardInforming = "informing"
+	boardBlocking  = "blocking"
+
+	layoutISO = "2006-01-02"
+)
+
+// ffCmd represents the base command when called without any subcommands
```

Comment:
Let's remove the comment or fix it by:
```suggestion
// testGridCmd …
```